README.md

```diff
@@ -0,0 +1,44 @@
+## Manim DSL Compiler
+
+ManimDSL is a domain specific language using Manim to create animations for
```

Comment:
Can we add a link to the Manim GitHub repo here?

Reply:
I've added the link to the whole repo and there's a dedicated Documentation section in the Table of Contents so it should be okay?

Reply:
Where is the link to the whole Manim repo?

Reply:
Oh you meant the whole Manim Repo... sure I'll add that in then